<launch>

    <arg name="publish_robot_source_odom_tf" default="true"/>

    <node name="base_only" pkg="rubber_navigation" type="baseOnly" output="screen">
        <param name="base_foot_print" value="base_link"/>
        <param name="odom_frame" value="odom"/>
        <param name="publish_tf" value="$(arg publish_robot_source_odom_tf)"/>
        <param name="serial_addr" value="/dev/ttyUSB0"/>
    </node>

    <include file="$(find rubber_navigation)/launch/common.launch">
        <arg name="publish_robot_source_odom_tf" value="$(arg publish_robot_source_odom_tf)"/>
    </include>
</launch>